frontend/src/components/Workspace/Visualize/VisualizeItem.tsx

```diff
@@ -160,11 +158,8 @@ const ItemHeader = memo(function ItemHeader({ itemId }: ItemIdProps) {
             : { itemId, deleteData: false },
         ),
       )
-      if (!roiFilePath || !workspaceId) return
-      dispatch(cancelRoi({ path: roiFilePath, workspaceId }))
-      //eslint-disable-next-line
     },
-    [workspaceId, roiFilePath],
+    [workspaceId],
```

Comment:
pls add `// eslint-disable-next-line react-hooks/exhaustive-deps`

Reply:
@ReiHashimoto I'm so sorry, I missed it. I have updated its dependencies.

Reply:
@sanglevinh
> I think this PR does not need to change the VisualizeItem.tsx file. I restored it.

I get it, thank you!